ServerlessBenchmark/TriggerTests/BaseTriggers/QueueTriggerTest.cs

```diff
@@ -174,40 +174,37 @@ await CloudPlatformController.EnqueueMessagesAsync(new CloudPlatformRequest()
         }
 
         private async Task<bool> VerifyQueueMessagesExistInTargetQueue(int expected)
```

Comment:
I agree this is different, but the behavior now matches the intended semantics. The method does not claim to consume the messages on the queue. We already have logic that clears the queues, see: https://github.com/Azure/azure-functions-performance-test/blob/master/ServerlessBenchmark/TriggerTests/BaseTriggers/QueueTriggerTest.cs#L22